action/clihelper_test.go

```diff
@@ -0,0 +1,41 @@
+package action
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_askForGitConfigUser(t *testing.T) {
+	s := New("test-init")
+	s.isTerm = true
+
+	_, _, err := s.askForGitConfigUser()
+	if err == nil {
+		t.Error("Did not return error")
+	}
+}
+
+func Test_askForGitConfigUserNonInteractive(t *testing.T) {
+	s := New("test-init")
+	// for explicitness
+	s.isTerm = false
+
+	keyList, err := s.gpg.ListPrivateKeys()
+	if err != nil {
+		t.Error(err.Error())
+	}
+
+	name, email, _ := s.askForGitConfigUser()
+
+	// tests cannot know whether keyList returned empty or not.
+	// a better distinction would require mocking/patching
+	// calls to s.gpg.ListPrivateKeys()
```

Comment:
This keyring is used indirectly in [init_test.go#22](https://github.com/justwatchcom/gopass/pull/185/commits/ff0860ed75e819ce1c684070e46969e3c3900817#diff-9e21dcdd5a0a12483a17f3cd3057ef75R22), but I couldn't figure out how to also leverage it in the unit test here.